doc/classes/Camera3D.xml

```diff
@@ -55,8 +55,18 @@
 			<argument index="0" name="world_point" type="Vector3">
 			</argument>
 			<description>
-				Returns [code]true[/code] if the given position is behind the camera.
+				Returns [code]true[/code] if the given position is behind the camera (the blue part of the below diagram).
 				[b]Note:[/b] A position which returns [code]false[/code] may still be outside the camera's field of view.
+				[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/camera3d_position_frustum.png]Camera3D is position methods[/url]
```

Comment:
I guess this doesn't work for the parser?
If so, maybe double quotes instead of `[code]`. It took me a while to understand what "Camera is position methods" referred to, I thought it was a typo/bogus grammar.
```suggestion
				[url=https://raw.githubusercontent.com/godotengine/godot-docs/master/img/camera3d_position_frustum.png]Camera3D [code]is_position[/code] methods[/url]
```